ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1518,6 +1518,64 @@ presubmits:
           name: prow-job-cluster-launch-installer-ovirt-e2e
         name: job-definition
     trigger: (?m)^/test( | .* )e2e-ovirt,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-steps
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-master-e2e-steps
+    optional: true
+    rerun_command: /test e2e-steps
+    skip_report: true
```

Comment:
I guess this is why it's not showing up in GitHub.  Personally, I don't see why anyone would ever want to set this.  Folks who aren't interested can just not click through.

Reply:
The idea is to avoid noise in the PRs in `openshift/installer` while we work on the job. The final one will not have it.